### PR TITLE
Added Patton SmartNode OS discovery and polling

### PR DESCRIPTION
**Pull request by PipoCanaja**
Closing issue https://github.com/librenms/librenms/issues/8454

capture d ecran 2019-03-04 a 23 09 15

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/patton-sn.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+os: patton-sn
+group: patton
+text: 'Patton SmartNode'
+type: network
+icon: patton
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_processor, text: 'Processor Usage' }
```
Comment (TheGreatDoc) [inline includes/definitions/patton-sn.yaml]: Maybe overs could be proc, temp and states(not sure about the last one)?

**Comment by TheGreatDoc** [comment]
Nice job 👍

File: includes/definitions/patton-sn.yaml (hunk 1 above)
Comment (PipoCanaja) [reply includes/definitions/patton-sn.yaml]: Mempools is here now. Had forgot about it. Device traffic is indeed useless on these. Count of calls makes more sense.

**Comment by PipoCanaja** [comment]
OK, this time, looks complete. We'll see how the test are going.

File: includes/polling/os/patton-sn.inc.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+//PATTON SmartNode
+$oids = array(
+    'hw_release' => '.1.3.6.1.4.1.1768.100.1.3.0',
```
Comment (TheGreatDoc) [inline includes/polling/os/patton-sn.inc.php]: You are going to hate me... But can you move this array() to []?

Comment (PipoCanaja) [reply includes/polling/os/patton-sn.inc.php]: No problem, I sometimes see those, but usually not :) On its way !

**Pull request merged**